domain/geo/distance_calculator.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+
+import great_circle_calculator.great_circle_calculator as gcc
+
+from domain.geo import Coordinates
+
+
+def calc_distance_in_kilometers(p1: Coordinates, p2: Coordinates) -> float:
+    distance_in_meters = gcc.distance_between_points(
+        (p1.longitude, p1.latitude),
+        (p2.longitude, p2.latitude),
+        unit="meters",
```

Comment:
Does the library allow to compute distancce directly in kilometers (or miles for US audience)? BTW, we will eventually need to make distance units configurable.

Reply:
Supported units are ['meters', 'kilometers', 'miles', 'feet', 'yards', 'nautical_miles']